pkg/cmd/cli/describe/printer.go

```diff
@@ -26,7 +26,7 @@ import (
 )
 
 var (
-	buildColumns            = []string{"NAME", "TYPE", "FROM", "STATUS", "STARTED"}
+	buildColumns            = []string{"NAME", "TYPE", "FROM", "STATUS", "STARTED", "REASON"}
```

Comment:
if this column will be empty in normal case, why we are showing it in oc get and not just in oc describe?